R/processing.R

```diff
@@ -161,6 +161,31 @@ totalise_data <- function(data) {
   return(data)
 }
 
+#' Default processing steps to run
+#' @description The default processing steps to which are always run. Runs on
+#' clean data
+#' @param data A data table
```

Comment:
is everything else exported or internal?

Reply:
all the other processing functions lack the export tag so just followed suit with these

Reply:
sounds sensible (don't forget the concept tag for docs) if missing.